### PR TITLE
CMake: changes to create symbolic link with playready

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -37,3 +37,8 @@ set_target_properties(${MODULE_NAME} PROPERTIES PREFIX "")
 
 install(TARGETS ${MODULE_NAME} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/${NAMESPACE}/OCDM)
 
+if(NOT "${CMAKE_FIND_ROOT_PATH}" STREQUAL "")
+    install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink /etc/playready  ${CMAKE_CURRENT_BINARY_DIR}/playready)")
+    install(DIRECTORY DESTINATION ${CMAKE_INSTALL_PREFIX}/../root/OCDM)
```
Comment (MFransen69) [inline CMakeLists.txt]: The /.../root needs to be replaced by the ${PERSISTENT_PATH} variable.

File: CMakeLists.txt [hunk 2]
```diff
@@ -37,3 +37,8 @@ set_target_properties(${MODULE_NAME} PROPERTIES PREFIX "")
...
+    install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/playready DESTINATION ${CMAKE_INSTALL_PREFIX}/../root/OCDM/)
```
Comment (MFransen69) [inline CMakeLists.txt]: See above also here of course /.../root needs to be replaced

File: CMakeLists.txt (hunk 1 above)
Comment (pawel-stanek-ml) [reply CMakeLists.txt]: Handling that already on sander/new-ocdm-interface

**Comment by pawel-stanek-ml** [comment]
Merged to sander/new-ocdm-interface and to master from there.

**Pull request closed without merging**